dataland-internal-storage/src/main/kotlin/org/dataland/datalandinternalstorage/controller/StorageController.kt

```diff
@@ -24,9 +25,11 @@ class StorageController(
         logger.info("Selecting data from database with (data id: $dataId) and (correlation id: $correlationId).")
         return ResponseEntity.ok(dataStore.selectDataSet(dataId))
     }
-
-    override fun insertData(correlationId: String, body: String): ResponseEntity<InsertDataResponse> {
-        logger.info("Inserting data into database with (correlation id: $correlationId).")
-        return ResponseEntity.ok(InsertDataResponse(dataStore.insertDataSet(body)))
+    override fun insertData(message: Message): ResponseEntity<InsertDataResponse> {
```

Comment:
I think we don't need this method at all.

Reply:
Will be gone with the next push

Reply:
ok